url.bs

```diff
@@ -2099,10 +2099,19 @@ string <var>input</var>, optionally with a <a>base URL</a> <var>base</var>, opti
        <li><p>If <var>url</var> <a>is special</a> and <a>c</a> is U+005C (\),
        <a>validation error</a>.
 
-       <li><p>If <var>buffer</var> is a <a>double-dot path segment</a>, <a>shorten</a>
-       <var>url</var>'s <a for=url>path</a>, and then if neither <a>c</a> is U+002F (/), nor
-       <var>url</var> <a>is special</a> and <a>c</a> is U+005C (\), <a for=list>append</a>
-       the empty string to <var>url</var>'s <a for=url>path</a>.
+       <li>
+        <p>If <var>buffer</var> is a <a>double-dot path segment</a>, then:
+
+        <ol>
+         <li><p><a>Shorten</a> <var>url</var>'s <a for=url>path</a>.
+
+         <li>
+          <p>If neither <a>c</a> is U+002F (/), nor <var>url</var> <a>is special</a> and <a>c</a> is
+          U+005C (\), <a for=list>append</a> the empty string to <var>url</var>'s
+          <a for=url>path</a>.
+
+          <p class="note no-backref">Usually <a>c</a> is U+002F (/) at this point.
```

Comment:
This doesn't really help much IMO. Addressing #482 would involve explaining the meaning of empty path segments, either here or somewhere else that this step could link to.

Reply:
How are they different from other path segments?

Reply:
Empty is often a special case, at least in human's minds. I agree for computers they are not different.

Reply:
Okay, would something like "This means that for input `/foo/..` the result is `/foo/` and not `/foo`." work?

Reply:
Yeah that makes a lot of sense to me.